From bad4bf39065bd3efdb95e0335b4c8b2b976902ff Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Thu, 16 May 2019 13:22:07 +0200
Subject: [PATCH] target/linux/mvebu: rename Turris Omnia target to be
 consistent

Master of OpenWRT renamed turris-omnia target to cznic_turris-omnia. We
are using this symbol in packages and to be consistend between Turris OS
4.x and 5.x we have to change that name here as well.
---
 target/linux/mvebu/image/cortex-a9.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/target/linux/mvebu/image/cortex-a9.mk b/target/linux/mvebu/image/cortex-a9.mk
index 6bc6269da6..e5605ed374 100644
--- a/target/linux/mvebu/image/cortex-a9.mk
+++ b/target/linux/mvebu/image/cortex-a9.mk
@@ -155,7 +155,7 @@ define Device/globalscale-mirabox
 endef
 TARGET_DEVICES += globalscale-mirabox
 
-define Device/turris-omnia
+define Device/cznic_turris-omnia
   KERNEL_INSTALL := 1
   KERNEL := kernel-bin
   KERNEL_INITRAMFS := kernel-bin
@@ -169,6 +169,6 @@ define Device/turris-omnia
   IMAGE_NAME = $$(2)
   DEVICE_DTS := armada-385-turris-omnia
 endef
-TARGET_DEVICES += turris-omnia
+TARGET_DEVICES += cznic_turris-omnia
 
 endif
-- 
2.21.0

